src/library.py

```diff
@@ -998,43 +1177,29 @@ def verify(self):
                 if (
                     day["O_sch"] <= tol_occ
                 ).all():  # confirmed this room is NOT rented out
-                    if (day["m_z_oa"] == 0).all():
-                        result_repo.append(1)  # pass,
-                    else:
-                        result_repo.append(0)  # fail
+                    if self.determine_30_min_vacancy(day["O_sch"], time_step):
+                        if (day["m_z_oa"] == 0).any():
+                            result_repo.append(True)  # pass
+                        else:
+                            result_repo.append(False)  # fail
                 else:  # room is rented out
                     if (day["m_z_oa"] > 0).all():
-                        if (
-                            day["m_z_oa"] == m_z_oa_set
-                            or day["m_z_oa"].sum(axis=1) == zone_volume
+                        if self.check_airflow_equals_airflow_set_for_hour(
+                            self.df["O_sch"], m_z_oa_set, time_step
+                        ) or self.check_sum_airflow_equals_volume_per_hour(
+                            self.df["m_z_oa"], self.df["area_z"], self.df["height_z"]
                         ):
-                            result_repo.append(1)  # pass
+                            result_repo.append(True)
+
                         else:
-                            result_repo.append(0)  # fail
+                            result_repo.append(False)
                     else:
-                        result_repo.append(0)
+                        result_repo.append(True)
                 year_info = day.index.year[0]
 
         dti = pd.date_range("2020-01-01", periods=365, freq="D")
         self.result = pd.Series(result_repo, index=dti)
```

Comment:
the same comment from another occasion applies here as well. it looks like this would fail if the simulation data does not cover 365 days. I suggest you get index from simulation data directly.